README.md

```diff
@@ -28,6 +28,10 @@ If you need to run serial mode against an ASG with an expected `desired_capacity
 ./bouncer serial -a hashi-use1-stag-worker-linux:3,hashi-use1-stag-worker-windows:2
 ```
 
+## Rolling
+
+Rolling is the same as serial, but does not decrement. This means `min_size`, `max_size`, and `desired_capacity` can all be the same value. This is useful when you need to migrate workloads. Take nomad for example, when rolling the ASG, we want to drain the node gracefully, but we need a new node available for them to migrate to. We don't want to migrate to a node that is about to be terminated because it's on an old version. This mode allows us to terminate an instance, let the ASG create a new one, and migrate the workload before moving on to the next instance. If we were to use canary mode, it will terminate too fast before the allocations have become healthy and cause a potential outage.
```

Comment:
I think maybe just clean this up a little too, as I mentioned elsewhere on this PR, we use canary mode with Nomad, and it works fine. You just have to setup your terminate hooks and heartbeats correctly.

Maybe just make this stanza less opinionated, and include only the first part?

```
Rolling is the same as serial, but does not decrement. This means `min_size`, `max_size`, and `desired_capacity` can all be the same value. This mode allows you to terminate an instance, let the ASG create a new one, and migrate the workload before moving on to the next instance.
```